pontoon/base/static/js/main.js

```diff
@@ -12,6 +12,16 @@ var Pontoon = (function (my) {
   return $.extend(true, my, {
 
     /*
+     * Remove duplicates from array
+     */
+    removeDuplicates: function (array) {
```

Comment:
I wonder if Set() wouldn't be better https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set